src/openrct2/drawing/Drawing.String.cpp

```diff
@@ -934,18 +933,18 @@ static void ttf_process_initial_colour(int32_t colour, text_draw_info* info)
     }
 }
 
-void ttf_draw_string(rct_drawpixelinfo* dpi, const_utf8string text, int32_t colour, int32_t x, int32_t y)
+void ttf_draw_string(rct_drawpixelinfo* dpi, const_utf8string text, int32_t colour, const ScreenCoordsXY& coords)
 {
     if (text == nullptr)
         return;
 
     text_draw_info info;
     info.font_sprite_base = gCurrentFontSpriteBase;
     info.flags = gCurrentFontFlags;
-    info.startX = x;
-    info.startY = x;
-    info.x = x;
-    info.y = y;
+    info.startX = coords.x;
+    info.startY = coords.y;
```

Comment:
Weird code previously

Reply:
I wouldn't be surprised if they were simply copy-and-paste errors. I wonder if it will make any noticeable change.

Reply:
I assumed so